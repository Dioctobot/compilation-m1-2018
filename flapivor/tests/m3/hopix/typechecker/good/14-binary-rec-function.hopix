def fst : Int * Int -> Int (x, y) = fst (y, x)

val main : Int = fst (3, 4)