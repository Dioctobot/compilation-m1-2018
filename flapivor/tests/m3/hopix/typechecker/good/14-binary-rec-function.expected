fst : Int * Int -> Int
main : Int
